a10_octavia/controller/worker/tasks/a10_database_tasks.py

```diff
@@ -95,7 +95,7 @@ def execute(self, amphora, loadbalancer, role, status=constants.ACTIVE):
             device_name=vthunder_id, username=username,
             password=password, ip_address=amphora.lb_network_ip,
             undercloud=False, axapi_version=axapi_version,
-            port=port, protocol=protocol,
+            port=443, protocol='https',
```

Comment:
This shouldn't be hardcoded.

Reply:
Agreed. We will be shifting this to our `a10-octavia.conf` file under `[vthunder]` section. 
Not sure if it should be in this PR.

Reply:
Addressed.